README.md

```diff
@@ -7,121 +7,147 @@ FlowEHR is a safe, secure &amp; cloud-native development &amp; deployment platfo
+
+For CI-deployed environments, you should check those environment files into source control so that your CICD pipelines can use them. Follow the [CI section](#ci) for more info.
+
+### Locally
+
+When working locally and with the `$ENVIRONMENT` env var unset, FlowEHR will look for a file called `config.local.yaml`. This is gitignored by default, so we can create one now for our own environment locally without worrying about accidentally checking it in:
 
 ```bash
-cp config.sample.yaml config.yaml
+cp config.sample.yaml config.local.yaml
 ```
 
-Then edit `config.yaml` and specify the following values:
+Once you've created it, specify the following values:
 
-- `suffix` - a suffix to apply to all deployed resources (i.e. `flowehr-uclh`)
-- `environment` - a unique name for your environment (i.e. `jgdev`)
+- `id` - a unique identifier to apply to all deployed resources (i.e. `myflwr`)
+- `environment` - a name for your environment (set this to `local`)
 - `location` - the [Azure region](https://azuretracks.com/2021/04/current-azure-region-names-reference/) you wish to deploy resources to
-- `arm_subscription_id` - the [Azure subscription id](https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id) you wish to deploy to
+- `core_address_space` (optional) - override the default core address space, e.g. `10.1.0.0/24`
+
+- `transform` (optional)
+    - `spark_version` (optional) - the Spark version to install on your local cluster and Databricks clusters
+    - `repositories` (optional) - list of Git repositories containing transform artifacts to clone and deploy to FlowEHR
+
+- `serve`
+    - `github_owner` - the GitHub organisation to deploy FlowEHR app repositories to
+    - `github_token` (local only) - a GitHub PAT for authenticating to GitHub. See the [apps README](./apps/README.md) for details.
 
-For the full reference of possible configuration values, see the [config schema file](./config_schema.json).
 
 ## Deploying
+
 ### Locally
 
 1. Log in to Azure
 
     Run `az login` to authenticate to Azure
 
-2. Run `make all`
+2. Select your subscription
 
-    To bootstrap Terraform, and deploy all infrastructure and apps, run
+    If you're deploying to a subscription that isn't your default, switch to it:
 
     ```bash
-    make all
+    az account set -s SUB_NAME_OR_ID
     ```
 
-    > For more info on configuring and deploying apps, see the [README](./apps/README.md)
+3. Run `make infrastructure`
 
-    Alternatively, you can deploy just infrastructure:
+    To bootstrap Terraform, and deploy all infrastructure, run
 
     ```bash
     make infrastructure
     ```
 
-    You can also deploy individual infrastructure modules, as well as destroy and other operations. To see all options:
+    Alternatively, if you just want to deploy transform infrastructure but not serve:
```

Comment:
would be really nice to link to a diagram here for the "what is serve" question?